src/Element/ContactTabContent.php

```diff
@@ -94,6 +95,12 @@ public static function preRenderTabContent(array $element) {
           'content' => $block->build(),
         ];
 
+        // Add edit link to title.
+        if (in_array(DashboardBlockInterface::class, class_implements($block))) {
```

Comment:
Would `$block implements Interface` be better?

Reply:
I'm pretty sure what you put isn't valid php :p 

I'm following core convention with my syntax. See:

Drupal\Core\Annotation\ContextDefinition::__construct()
Drupal\Core\Entity\TypedData\EntityDataDefinition::getPropertyDefinitions()

Reply:
(except that they tend to write out the full namespace of the class as a string, which feels like overkill...)

Reply:
Sorry, I meant `instanceof`.